plugins/modules/luks_device.py

```diff
@@ -170,6 +170,47 @@
             - "Will only be used on container creation."
         type: str
         version_added: '1.1.0'
+    pbkdf:
+        description:
+            - "This option allows the user to configure the Password-Based Key Derivation
+               Function (PBKDF) used."
+            - "Will only be used on container creation, and when adding keys to an existing
+               container."
```

Comment:
```suggestion
            - This option allows the user to configure the Password-Based Key Derivation
               Function (PBKDF) used.
            - Will only be used on container creation, and when adding keys to an existing
               container.
```
I'd avoid unnecessary things. It's just a suggestions, if disagree, feel free to mark as resolved and ignore